js/blocks/controls/checkbox.js

```diff
@@ -18,9 +16,6 @@ const BlockLabCheckboxControl = ( props, field, block ) => {
 				attr[ field.name ] = checkboxControl
 				setAttributes( attr )
 			}}
-			onBlur={
```

Comment:
This is removed entirely since it seemed like overkill to fetch a new preview via the REST API every time a value changed.